text/103-leaving-x-twitter.md

```diff
@@ -0,0 +1,68 @@
+---
+Title: Transitioning from X/Twitter to new social media platforms
+Drivers: Théo Zimmermann (@Zimmi48)
+---
+
+## Summary
+
+This RFC proposes that as part of the actions to be taken in the context of the renaming of Coq to the Rocq Prover, a departure from X/Twitter should be announced and implemented. The rationale for this decision is based on the changing ownership and policy at X/Twitter, the current management of the X/Twitter account, the overall context of many academics and academic institutions leaving X/Twitter, and the opportunity presented by the renaming to move to more appropriate platforms.
+
+## Context and motivation
+
+The decision to leave X/Twitter is motivated by several factors:
+
+1. **Changing ownership and policy at X/Twitter:**
+  Since the acquisition of Twitter by Elon Musk, significant policy changes, including in moderation, promotion of misinformation, etc. have caused concern within the scientific and open-source communities. The Coq/Rocq team's values of openness, collaboration, and inclusivity, but also the academic and scientific communities' values, are increasingly at odds with the direction of the platform.
```

Comment:
the cost of free speech is that there will be some misinformation. the censorship of often non-official viewpoints (especially in the sciences with shaky foundations, e.g. social/medical sciences) on other platforms have also caused a lot of concern amongs scientists, but most of those scientists do not speak up.
(I am not making a claim whether X fully suports its goals of free speech, but rather claiming that that the claimed goal is a worthy goal vs concerns over spread of "misinformation". who decides what is misinformation?)

Reply:
I think it's a mistake for Rocq to act for political reasons.  Boycotts and censorship are inconsistent with openness and inclusivity--openness to other ideas and inclusivity of those with differing viewpoints.  Misinformation is a problem throughout the media.  Has there been misinformation about Rocq?

FWIW, it's also becoming evident, at least in the US, that the general public strongly disagrees with some of the values of the academic and scientific communities.  You may recall the recent resignations of several university presidents (of Harvard, University of Pennsylvania and Cornell).

Reply:
Maybe also consider: How do you really know that using Twitter is against the values of the academic and scientific communities?  There have been some notorious cases recently in the US of faculty being forced out of their positions because they openly disagreed with the dominant political narrative established by those who have the power at the universities.